test/jdk/jdk/jfr/event/runtime/TestResidentSetSizeEvent.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is the `2022, 2023` intentional or should it just be `2023`?

Reply:
It was intentional since I copy-n-pasted the structure of the code from the NMT JFR test.